packages/react-accordion/src/components/AccordionHeader/AccordionHeader.types.ts

```diff
@@ -46,7 +46,7 @@ export type AccordionHeaderCommons = {
   inline: boolean;
 };
 
-export type AccordionHeaderProps = ComponentProps<AccordionHeaderSlots> & Partial<AccordionHeaderCommons>;
+export type AccordionHeaderProps = ComponentProps<Partial<AccordionHeaderSlots>> & Partial<AccordionHeaderCommons>;
```

Comment:
I haven't seen this in PR's description. Can you please clarify why `Partial<>` is used sometimes for `ComponentProps`?

This does not look intuitive. Can we avoid it somehow?

Reply:
Added to the PR description:
This was required because ComponentProps effectively had `Partial<Slots>` built-into it before this change (it added `?` to every slot). After this change, it just includes the `Slots` type directly.
While I could have added an explicit Partial<> inside ComponentProps, that was a bad idea anyways, because it made it difficult to have a required slot prop. Instead, each component can add Partial to the slots if it makes sense in that case.

Reply:
It's along the same lines as `Partial<AccordionHeaderCommons>` next to it.